src/models/storage.ts

```diff
@@ -617,29 +614,53 @@ export class EntryStorage {
       }
 
       if (!entryData.type) {
-        entryData.type = OTPType[OTPType.totp];
+        entryData.type = OTPType.totp;
```

Comment:
move this into the case statement so we don't have to cast to unknown below. we can set `type: unknown` in `otp.d.ts`/`RawOTPStorage`